cohortApp/routes/cohorts.js

```diff
@@ -0,0 +1,170 @@
+const express = require("express");
+const router = express.Router();
+
+const queries = require("../db/queries");
+
+const isValidId = (req, res, next) => {
+  if (!isNaN(req.params.id)) return next();
+  res.status(400);
+  next(new Error("Invalid Id"));
+};
+
+const validCohort = cohort => {
+  const hasName = typeof cohort.name === "string" && cohort.name.trim() !== "";
+  const hasMembers = typeof cohort.members === "string" && cohort.members.trim() !== "";
```

Comment:
To keep organized, I would put this and other function expressions into their own module and require them at the top. That way, your cohort router file will only contain code for route handlers related to cohorts.